problem_builder/v1/upgrade.py

```diff
@@ -42,14 +43,18 @@ def upgrade_block(block):
     Given a MentoringBlock "block" with old-style (v1) data in its "xml_content" field, parse
     the XML and re-create the block with new-style (v2) children and settings.
     """
-    assert isinstance(block, MentoringBlock)
+    assert isinstance(block, (MentoringBlock, NewMentoringBlock))
     assert bool(block.xml_content)  # If it's a v1 block it will have xml_content
     store = block.runtime.modulestore
     xml_content_str = block.xml_content
     parser = etree.XMLParser(remove_blank_text=True)
```

Comment:
@bradenmacdonald nevermind, I've checked it out and it indeed works just fine with comments